src/server/auth/server/oidc.go

```diff
@@ -118,6 +63,31 @@ func CryptoString(n int) string {
 	return base64.RawURLEncoding.EncodeToString(b)
 }
 
+// validateOIDC validates an OIDC configuration before it's stored in etcd.
+func validateOIDCConfig(config *auth.AuthConfig) error {
```

Comment:
This was the only validation we were doing in `config.go` for OIDC, so I pulled it out here.